### PR TITLE
feat(as-116): add domain and email to lpa list

### PR DESCRIPTION
**Pull request by rcmsoftwaredevelopment**
add domain and email to lpa list

## Ticket Number
https://pins-ds.atlassian.net/browse/AS-116

## Description of change
add domain and email to lpa list

## Checklist

- [ ] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [ ] My commit history in this PR is linear
- [ ] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `master` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: packages/appeals-service-api/src/schemas/lpa.spec.js [hunk 1]
```diff
@@ -236,12 +236,12 @@ describe('LPA schema test', () => {
...
-390,W43000018,Cardiff LPA,315270,178887,-3.22212,51.50254,142287245.75211,79678.4715379399
-2,E60000002,Darlington LPA,428029,515648,-1.56835,54.53534,197475688.992432,107206.401677287
-331,N13000004,Belfast LPA,146465,529747,-5.92535,54.59853,137716630.472664,65908.6538260691
-343,S44000005,City of Edinburgh LPA,320193,669416,-3.27826,55.91119,263351803.906158,108121.329976751`;
+        const lpaExample = `OBJECTID,LPA19CD,LPA19NM,email,domain
+1,E60000001,County Durham LPA,,
+390,W43000018,Cardiff LPA,,
+2,E60000002,Darlington LPA,,
+331,N13000004,Belfast LPA,,
+343,S44000005,City of Edinburgh LPA,,`;
```
Comment (a6software) [inline packages/appeals-service-api/src/schemas/lpa.spec.js:244]: worth adding extra tests for when `email` and / or `domain` are specified, e.g.

`68,E60000068,Bradford,appealsplanning@bradford.gov.uk,bradford.gov.uk`

?

**Review by a6software — COMMENTED** [review]
I don't understand the purpose of this, *but* when I updated the lpa-list.csv, I also needed to do this: 

https://github.com/Planning-Inspectorate/appeal-planning-decision/pull/751

It's worse than it looks - largely copy / paste, run through that url, and paste it back in. I've documented this somewhere but can't for the life of me find where.

File: packages/appeals-service-api/src/schemas/lpa.spec.js (hunk 1 above)
Comment (rcmsoftwaredevelopment) [reply packages/appeals-service-api/src/schemas/lpa.spec.js:244]: I'll sort this shortly

Comment (brunswickcomputing) [reply packages/appeals-service-api/src/schemas/lpa.spec.js:244]: i think this is where your documentation is Chris
https://pins-ds.atlassian.net/wiki/spaces/AAPDS/pages/457703525/DRAFT+Operational+Support+Manual

Comment (a6software) [reply packages/appeals-service-api/src/schemas/lpa.spec.js:244]: I have moved this into a sub document of that Operational Support Manual page:

https://pins-ds.atlassian.net/wiki/spaces/AAPDS/pages/632487948/Eligible+Local+Planning+Departments

File: packages/appeals-service-api/src/schemas/lpa.js [hunk 2]
```diff
@@ -74,11 +73,14 @@ module.exports = class LPA {
     return (await csvParser(lpaList))
       .filter(({ LPA19CD }) => {
         /* Filter out Scotland and Northern Ireland LPAs */
-        const include = LPA19CD.startsWith('S') === false && LPA19CD.startsWith('N') === false;
+        const include =
+          LPA19CD.startsWith('S') === false &&
+          LPA19CD.startsWith('N') === false &&
+          LPA19CD.startsWith('W') === false;
         logger.trace({ include, id: LPA19CD }, 'Filtering out Scottish and Northern Irish LPAs');
```
Comment (a6software) [inline packages/appeals-service-api/src/schemas/lpa.js:80]: and I'm guessing 'W'elsh?

**Comment by a6software** [comment]
Checked the LPA encode / decode, seems good 👍

**Comment by a6software** [comment]
Is this ready for review btw? (missing the label if so - just curious)

File: packages/appeals-service-api/src/schemas/lpa.js (hunk 2 above)
Comment (rcmsoftwaredevelopment) [reply packages/appeals-service-api/src/schemas/lpa.js:80]: It is.  I will extend that comment.  Seems the original file contained Welsh lpa as well, but this new file only contains English ones.

**Review by a6software — APPROVED** [review]

**Review by AnthonyJWalsh — APPROVED** [review]

**Pull request merged**